### ISSUE TITLE
documentation out of date for m1 macs

### ISSUE THREAD

**Issue by jhgorse** [Parallels/vagrant-parallels#419]
https://parallels.github.io/vagrant-parallels/docs/getting-started.html

```
brew install vagrant
vagrant plugin install vagrant-parallels

vagrant init bento/ubuntu-18.04
vagrant up --provider=parallels
```

results in
```
joe.gorse@Joes-MacBook-Pro ~ % vagrant init bento/ubuntu-18.04
A `Vagrantfile` has been placed in this directory. You are now
ready to `vagrant up` your first virtual environment! Please read
the comments in the Vagrantfile as well as documentation on
`vagrantup.com` for more information on using Vagrant.
joe.gorse@Joes-MacBook-Pro ~ % vagrant up --provider=parallels
Bringing machine 'default' up with 'parallels' provider...
==> default: Box 'bento/ubuntu-18.04' could not be found. Attempting to find and install...
    default: Box Provider: parallels
    default: Box Version: >= 0
==> default: Loading metadata for box 'bento/ubuntu-18.04'
    default: URL: https://vagrantcloud.com/bento/ubuntu-18.04
==> default: Adding box 'bento/ubuntu-18.04' (v202112.19.0) for provider: parallels
    default: Downloading: https://vagrantcloud.com/bento/boxes/ubuntu-18.04/versions/202112.19.0/providers/parallels.box
==> default: Successfully added box 'bento/ubuntu-18.04' (v202112.19.0) for 'parallels'!
==> default: Registering VM image from the base box 'bento/ubuntu-18.04'...
==> default: Creating new virtual machine as a linked clone of the box image...
==> default: Unregistering the box VM image...
==> default: Setting the default configuration for VM...
==> default: Checking if box 'bento/ubuntu-18.04' version '202112.19.0' is up to date...
==> default: Setting the name of the VM: joegorse_default_1648672364231_73279
==> default: Preparing network interfaces based on configuration...
    default: Adapter 0: shared
==> default: Clearing any previously set network interfaces...
==> default: Booting VM...
There was an error while command execution. The command and stderr is shown below.

Command: ["/usr/local/bin/prlctl", "start", "ab8b7509-39df-4d69-8f07-e7415d7c0d29"]

Stderr: Failed to start the VM: Unable to start "joegorse_default_1648672364231_73279" because your Mac is equipped with the Apple M1 chip that does not support Intel-based operating systems. To create a compatible virtual machine, use an ISO or VHDX image file with an ARM-based operating system.
```

At the time of this reading, there are no available aarch64 or arm64v8 for ubuntu 18. Consider updating the documentation and providing an aarch64 box to start with.

Cheers,
Joe

**Comment by rcousens** [comment]
Use ubuntu 20.04, bento provides an arm64 box for focal but not bionic.

See https://github.com/chef/bento/tree/main/packer_templates/ubuntu

**Comment by jhgorse** [comment]
Right, so let's update the documentation 
https://parallels.github.io/vagrant-parallels/docs/getting-started.html

to reflect something that'll work for both x86 and arm64 platforms by default, like 20.04.

**Comment by NahsiN** [comment]
Hello I've and m1 machine running Vagrant 2.2.19. The following still does not work for me. It gives the _same_ error reported in the first post
```
vagrant init bento/ubuntu-20.04
vagrant up --provider=parallels
```

**Comment by sai-kumar-peddireddy** [comment]
@legal90 we can close this issue. I think

**Comment by legal90** [comment]
Fixed by #436 & #438
